### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by abenaTenkorang**
**In this pull request, I implement the following**

- Add a new JavaScript file and import it as a module:
- it will contain methods related to the status updates (completed: true / false).
- Add event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Implement a function for the "Clear all completed" button (use filter() method).
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @abenaTenkorang 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a _**README.md**_ file. Kindly use this [template](https://github.com/microverseinc/readme-template). When using the template:
- Kindly remove the logo
- Fill most sections heders with the relevant information
- Work on the MIT link also. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-loop-func */
```
Comment (rustam-tolipov) [inline src/index.js]: **You did a great job, all the functions are working as expected.**

- [ ] However, we got one issue that needs to be fixed. In this project, students are not allowed to use eslint disabler. 
Kindly fix the issues and remove the eslint disabler.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @abenaTenkorang,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (abenaTenkorang) [reply src/index.js]: @Rustamxon7 Please help elaborate on the issue. Thank you

Comment (rustam-tolipov) [reply src/index.js]: Hello, @abenaTenkorang Sorry for the late reply...

Comment (rustam-tolipov) [reply src/index.js]: At that point. You have used no-loop-func disabling that is, you shouldn't declare a function inside the loop. I suggest you create all declared functions outside of the loop and then you can reuse them inside of the loop again.

Comment (abenaTenkorang) [reply src/index.js]: Issue resolved. Thank you @Rustamxon7

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @abenaTenkorang, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**